src/dask_awkward/lib/core.py

```diff
@@ -1965,7 +1965,16 @@ def __init__(self, the_fn, arg_repackers, kwarg_repacker, arg_lens_for_repackers
         self.kwarg_repacker = kwarg_repacker
         self.arg_lens_for_repackers = arg_lens_for_repackers
 
-    def __call__(self, *args_deps_expanded):
+    def __repr__(self):
+        return (
+            repr(self.fn)
+            .replace("<", "")
+            .replace(">", "")
+            .replace("function ", "")
+            .replace("built-in ", "")
```

Comment:
Fortunately this doesn't get called a lot - or not at all on the compute path - but it's pretty annoying! 

Does it amount to `f"{self.fn.__module__}.{self.fn.__qualname__}-repacked"` ?

Reply:
ah yeah it's this but without the `__module__` component. Thanks!